pkg/terminal/command.go

```diff
@@ -1032,13 +1051,28 @@ func (c *Commands) next(t *Term, ctx callContext, args string) error {
 	if c.frame != 0 {
 		return notOnFrameZeroErr
 	}
-	state, err := exitedToError(t.client.Next())
-	if err != nil {
-		printcontextNoState(t)
+	var count int64
+	var err error
+	if count, err = parseOptionalCount(args); err != nil {
 		return err
+	} else if count <= 0 {
+		return errors.New("Invalid next count")
 	}
-	printcontext(t, state)
-	return continueUntilCompleteNext(t, state, "next")
+	for ; count > 0; count-- {
+		state, err := exitedToError(t.client.Next())
+		if err != nil {
+			printcontextNoState(t)
+			return err
+		}
+		// If we're about the exit the loop, print the context.
+		if count == 1 {
```

Comment:
nit: let's just store the result of this calculation since we use it twice, something like `finishedNext := count == 1`.

Reply:
Sure -- your code base, your style!